activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -777,7 +777,9 @@ def add_foreign_key(from_table, to_table, options = {})
         execute schema_creation.accept(at)
       end
 
-      # Removes the given foreign key from the table.
+      # Removes the given foreign key from the table. The options parameters will be
```

Comment:
Is the options hash really ignored? I'm thinking about a scenario like: `remove_foreign_key :table, name: "my_fk"`